game/views.py

```
@@ -1,1157 +1,1172 @@
+@require_GET
+def check_username(request):
+    """Check if a username is already taken."""
+    username = request.GET.get('username', '').strip()
+    if not username:
+        return JsonResponse({'available': False, 'error': 'No username provided'}, status=400)
+    exists = User.objects.filter(username__iexact=username, is_active=True).exists()
+    return JsonResponse({'available': not exists})
+
+
+
+def register_view(request):
```

Ruff (0.15.14)

[warning] Too many branches (13 > 12)

(PLR0912)